addons/mail/tests/discuss/test_guest_feature.py

```diff
@@ -34,11 +34,12 @@ def test_channel_seen_as_guest(self):
         self.assertEqual(guest_member.seen_message_id, channel.message_ids[0])
 
     def test_subscribe_to_guest_channel(self):
+        self.env.cr.precommit.run()
```

Comment:
It's a bit annoying to have this appear in many tests, but I guess it's an ok price to pay for the optimization.